lib/chef/provider/execute.rb

```diff
@@ -78,6 +78,18 @@ def sensitive?
         !!new_resource.sensitive
       end
 
+      def live_stream?
+        !!new_resource.live_stream
```

Comment:
:+1: on @danielsdeleo thinking. #live_stream? should be true in those cases. Then the conditional below around new line 104 changes to:

```
if (Chef::Log.info? || live_stream?) && !sensitive?
  if run_context.events.formatter?
    opts[:live_stream] = Chef::EventDispatch::EventsOutputStream.new(run_context.events, :name => :execute)
  elsif stream_to_stdout?
    opts[:live_stream] = STDOUT
  end
end
```